src/chrome/content/rules/Wikimedia.xml

```diff
@@ -98,35 +129,32 @@ Fetch error: http://icinga.wmflabs.org/ => https://icinga.wmflabs.org/: (6, 'Cou
 
 	<rule from="^http://noboard\.chapters\.wikimedia\.org/"
 		to="https://noboard-chapters.wikimedia.org/" />
+		<test url="http://noboard.chapters.wikimedia.org/" />
 
-	<rule from="^http://(?:dataset2|download|sitemap)\.wikimedia\.org/"
+	<rule from="^http://(?:download|sitemap)\.wikimedia\.org/"
 		to="https://dumps.wikimedia.org/" />
+		<test url="http://download.wikimedia.org/" />
+		<test url="http://sitemap.wikimedia.org/" />
 
 	<rule from="^http://links\.email\.donate\.wikimedia\.org/"
 		to="https://www.links.mkt41.net/" />
+		<test url="http://links.email.donate.wikimedia.org/" />
 
-	<rule from="^http://(?:labs-ns[01]|virt0)\.wikimedia\.org/"
+	<rule from="^http://labs-ns0\.wikimedia\.org/"
 		to="https://wikitech.wikimedia.org/" />	
+		<test url="http://labs-ns0.wikimedia.org/" />
 
 	<rule from="^http://arbcom\.(de|en|fi|nl)\.wikipedia\.org/"
 		to="https://arbcom-$1.wikipedia.org/" />
+		<test url="http://arbcom.de.wikipedia.org/" />
+		<test url="http://arbcom.en.wikipedia.org/" />
+		<test url="http://arbcom.fi.wikipedia.org/" />
+		<test url="http://arbcom.nl.wikipedia.org/" />
 
 	<rule from="^http://download\.wikipedia\.org/"
 		to="https://dumps.wikimedia.org/" />
+		<test url="http://download.wikipedia.org/" />
 
-	<rule from="^http://wg\.en\.wikipedia\.org/"
-		to="https://wg-en.wikipedia.org/" />
```

Comment:
This is an old url (that still resolves) but would've borked due to the wrong cert if you visit https://wg.en.wikipedia.org